frontend/src/tests/pages/PlayPage.test.js

```diff
@@ -65,18 +54,62 @@ describe("PlayPage tests", () => {
             </QueryClientProvider>
         );
 
-        expect(await screen.findByTestId("buy-cow-button")).toBeInTheDocument();
-        const buyCowButton = screen.getByTestId("buy-cow-button");
-        fireEvent.click(buyCowButton);
+        expect(await screen.findByTestId("buy-a-cow-button")).toBeInTheDocument();
 
+        const buyMaxCowsButton = screen.getByTestId("buy-max-cows-button");
+        fireEvent.click(buyMaxCowsButton);
         await waitFor(() => expect(axiosMock.history.put.length).toBe(1));
+        const buyTenCowsButton = screen.getByTestId("buy-10-cows-button");
+        fireEvent.click(buyTenCowsButton);
+        await waitFor(() => expect(axiosMock.history.put.length).toBe(2));
+        const buyCowButton = screen.getByTestId("buy-a-cow-button");
+        fireEvent.click(buyCowButton);
+        await waitFor(() => expect(axiosMock.history.put.length).toBe(3));
 
-        const sellCowButton = screen.getByTestId("sell-cow-button");
+        const sellCowButton = screen.getByTestId("sell-a-cow-button");
         fireEvent.click(sellCowButton);
-
-        await waitFor(() => expect(axiosMock.history.put.length).toBe(2));
+        await waitFor(() => expect(axiosMock.history.put.length).toBe(4));
+        const sellTenCowsButton = screen.getByTestId("sell-10-cows-button");
+        fireEvent.click(sellTenCowsButton);
+        await waitFor(() => expect(axiosMock.history.put.length).toBe(5));
+        const sellAllCowsButton = screen.getByTestId("sell-all-cows-button");
+        fireEvent.click(sellAllCowsButton);
+        await waitFor(() => expect(axiosMock.history.put.length).toBe(6));
     });
 
+    // Not sure how to test this since I'm unable to call the onBuy and onSell functions from PlayPage
```

Comment:
If you aren't sure how to test this, then ask for help; but if you aren't sure, then this should be a draft PR, not a PR that is being reviewed by team members.

Commented out code doesn't go into the main branch